i18n/en.json

```diff
@@ -468,6 +468,10 @@
     "translation": "collapse"
   },
   {
+    "id": "api.command_collapse.success",
+    "translation": "Image links now collapse by default"
```

Comment:
Should the system message be `Image previews now collapse by default`? @it33 

Or is `image links` the correct terminology?